experiments/depth-series/depth_session.py

```
@@ -0,0 +1,112 @@
+              "context_capacity": cfg["context_capacity"], "max_tokens_per_turn": MAX_TOKENS_PER_TURN, "turns": []}
+    try:
+        open(decl, "w").write(patched(original, cfg)); report["declaration_sha256_as_run"] = sha(decl)
+        print("load:", base.admin(cfg, "load"), flush=True)
+        if not base.wait_socket(cfg): raise SystemExit("the gate socket never stood")
+        for i, group in enumerate(turns, 1):
+            body = "\n\n".join(open(p).read() for p in group)
+            t0 = time.time(); answer = base.gate_turn(cfg, f"{ASK}\n\n{body}", timeout=900); dt = time.time() - t0
+            entry = {"turn": i, "pages": [os.path.basename(p) for p in group], "input_words": len(body.split()),
+                     "seconds": round(dt, 1), "answer_kind": answer.get("kind"), "answer_chars": len(json.dumps(answer))}
+            report["turns"].append(entry); print(json.dumps(entry), flush=True)
+            if answer.get("kind") not in ("answered", "answer", None) and "text" not in json.dumps(answer): print("  answer:", str(answer)[:200], flush=True)
```

Ruff (0.16.3)

[error] Multiple statements on one line (colon)

(E701)